modules/las/src/lib/laslaz-decoder.ts

```diff
@@ -245,6 +306,10 @@ class LAZLoader {
     }
   }
 
+  /**
+   * Deletes the instance if it equals to null
```

Comment:
Confusing/incorrect comment
```suggestion
   * Deletes the instance
```